app/views/rewards/_packing_slip.html.erb

```diff
@@ -23,13 +23,11 @@
   <div class="halves ">
     <div>
       <p class="underline">Address</p>
-      <p><%= pledger.name%><br />
-         <%= pledger.perm_address%><br />
-      <% if pledger.perm_country == 'USA' %>
-        <%= pledger.perm_city%>, <%= pledger.perm_state%> <%= pledger.perm_zip %></p>
-      <% else %>
-        <%= pledger.perm_city %>, <%= pledger.perm_country %></p>
-      <% end %>
+      <p><%= pledger.name%><br>
+         <%= pledger.perm_address%><br>
+         <%= pledger.perm_city%>, <%= pledger.perm_state%> <%= pledger.perm_zip%>
```

Comment:
I don’t think this change is right. State and zip are not present when the country is not USA. Only city and country. (I might not have been consistent in this, but I believe that is how it should be)